claat/render/md.go

```diff
@@ -159,11 +164,7 @@ func (mw *mdWriter) url(n *types.URLNode) {
 	if n.URL != "" {
 		mw.writeString("[")
 	}
-	for _, cn := range n.Content.Nodes {
-		if t, ok := cn.(*types.TextNode); ok {
-			mw.writeString(t.Value)
-		}
-	}
+	mw.write(n.Content.Nodes...)
```

Comment:
Is this to make the "download" button a non-empty link text?

Reply:
relly ??